### ISSUE TITLE
TypeError: Cannot read property 'slice' of null

### ISSUE THREAD

**Issue by Willy5** [sandstorm-io/sandstorm#2981]
I try to restart sandstorm with "service sandstorm restart" but now the loading is infinite and in my log i have this error "TypeError: Cannot read property 'slice' of null" 

![sans titre 12](https://user-images.githubusercontent.com/5772618/30247707-362db804-961a-11e7-8f8b-7071b1f327f6.png)

**Comment by Willy5** [comment]
Found ! Don't fill base url with ip server, meteor check the "http://" with the domain name url it's ok !

**Comment by zenhack** [comment]
Glad you got past this hurdle getting set up. We should probably re-open this though, since Sandstorm should probably be validating the input and reporting a more useful error message, so there is actually a bug.

//cc @kentonv

**Comment by kentonv** [comment]
Yes, this is a bug, in that we should improve the error message...
